src/__tests__/table.scan.unit.test.ts

```diff
@@ -176,18 +179,16 @@ describe('scan', () => {
   })
 
   it('transforms a set into an array when parse is true', async () => {
-    DocumentClient.scan = jest.fn().mockReturnValueOnce({
-      promise: jest.fn().mockResolvedValueOnce({
-        Items: [
-          {
-            pk: 'test-pk',
-            sk: 'test-sk',
-            testSet: DocumentClient.createSet(['test1', 'test2']),
-            _et: 'TestEntity'
-          }
-        ],
-        LastEvaluatedKey: null
-      })
+    DocumentClient.send = jest.fn().mockReturnValueOnce({
```

Comment:
I would suggest here to use `before` or `beforeAll` with the mocks, since they are similar and could be reused